packages/odyssey/src/scss/components/_dropdown.scss

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-.has-dropdown {
+.had-ods-dropdown {
```

Comment:
Doesn't need to be this PR - but we should get into the habit of writing these kinds of changes down in a Migration Guide. Since this one is a bit different than the others (`has` -> `had`), consumers of this package can easily do a find + replace.

Reply:
Ah, this was a typo!